app/models/kandidatenlijst.js

```diff
@@ -1,16 +1,17 @@
 import Model, { attr, belongsTo, hasMany } from '@ember-data/model';
 
-export default Model.extend({
-  uri: attr(),
-  lijstnaam: attr(),
-  lijstnummer: attr(),
-  rechtstreekseVerkiezing: belongsTo('rechtstreekse-verkiezing', { inverse: 'heeftLijst' }),
-  kandidaten: hasMany('persoon', { inverse: 'isKandidaatVoor' }),
-  rdfaBindings: {// eslint-disable-line ember/avoid-leaking-state-in-ember-objects
+export default class KandidatenlijstModel extends Model {
+  @attr uri;
+  @attr lijstnaam;
+  @attr lijstnummer;
+  @belongsTo('rechtstreekse-verkiezing', { inverse: 'heeftLijst' }) rechtstreekseVerkiezing;
+  @hasMany('persoon', { inverse: 'isKandidaatVoor' }) kandidaten;
+
+  rdfaBindings = {
     lijstnaam: "http://www.w3.org/2004/02/skos/core#prefLabel",
     lijstnummer: "http://data.vlaanderen.be/ns/mandaat#lijstnummer",
     kandidaten: "http://data.vlaanderen.be/ns/mandaat#heeftKandidaat",
     rechtstreekseVerkiezing: "http://data.vlaanderen.be/ns/mandaat#behoortTot",
     "lijsttype": "http://data.vlaanderen.be/ns/mandaat#lijsttype"
```

Comment:
The model doesn't have an attribute / relationship named like this, can this be removed?

Reply:
it seems like it does exist in the backend: https://github.com/lblod/app-gelinkt-notuleren/blob/f3b160e22bb63b239a728a2477c34411057f8d41/config/resources/slave-mandaat-domain.lisp#L47 so we either add the relationship or remove it from the bindings. both are fine for me

Reply:
I removed the binding since the `lijsttype` model doesn't exist yet in the frontend either.